recipes/powerfit/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "powerfit" %}
+{% set org = "tdegeus" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/{{ org }}/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: d8dae7c58a8d188a3757ff1d9301beb6be8004ad1ba204efc64d726c1c5cb73b
+
+build:
+  number: 0
+  noarch: python
+  script_env: SETUPTOOLS_SCM_PRETEND_VERSION={{ version }}
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3
+    - setuptools_scm
+  run:
+    - scipy
+    - numpy
+    - python >=3
+
+test:
+  imports:
+    - {{ name }}
+  commands:
+    - pip check
+    - python -c "import {{ name }}; assert {{ name }}.version == '{{ version }}'"
+  requires:
+    - pip
+
+about:
+  home: https://github.com/{{ org }}/{{ name }}
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: Simple fit of a power-law to data
+  description: Simple fit of a power-law to data
+  doc_url: https://{{ name }}.readthedocs.io
+  dev_url: https://github.com/{{ org }}/{{ name }}
```

Comment:
The excessive use of jinja variables here don't make the recipe better, don't make them portable, and don't help much. But it does obfuscate a lot if someone is trying to read it. Can you please use jinja only for the name<->url and the version number?